edge/services/cpu_percent/horizon/userinput.json

```diff
@@ -1,11 +1,11 @@
 {
-    "global": [],
     "services": [
         {
             "org": "$HZN_ORG_ID",
-            "url": "$MYDOMAIN.$CPU_NAME",
-            "versionRange": "[0.0.0,INFINITY)",
-            "variables": {}
+            "url": "$SERVICE_NAME",
```

Comment:
I didn't think about this before when you asked me about this file, but i don't think this service will ever need any input variables, so you could remove this file altogether. (We try to encourage lower-level services to not require any input variables, because otherwise higher-level services that use them would have to add variables to their user input file before they could be registered, and that wouldn't be obvious that they need to do that. Plus a service like that is used via its rest api, could take any config values it needs via rest api parameters from the high-level service.